APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.m

```diff
@@ -310,6 +309,18 @@ - (void)signInOnCompletion:(void (^)(NSError *))completionBlock
                             self.sharingScope = APCUserConsentSharingScopeNone;
                         }
                     }
+                    
+                    NSArray *dataGroups = responseDictionary[@"dataGroups"];
+                    self.dataGroups = dataGroups;
+                    
+                    // TODO: Handle multiple consent groups with separate sub populations
+                    NSDictionary *consentStatuses = responseDictionary[@"consentStatuses"];
+                    for (id key in consentStatuses) {
+                        NSDictionary *status = [consentStatuses objectForKey:key];
```

Comment:
that being said, would be pretty simple to prioritize required subpopulations rather than 'last one wins'.